php/ext/google/protobuf/php_protobuf.h

```diff
@@ -0,0 +1,40 @@
+// Protocol Buffers - Google's data interchange format
+// Copyright 2008 Google Inc.  All rights reserved.
```

Comment:
// Copyright 2022 Google LLC

is the new form we use here

Reply:
fixed thanks